xml_converter/src/xml_converter.cpp

```diff
@@ -281,9 +270,16 @@ int main(int argc, char* argv[]) {
         }
         else if (!strcmp(argv[i], "--allow-duplicates")) {
             allow_duplicates = true;
+            arg_target = nullptr;
```

Comment:
We should probably change the default arg_target from input taco path to nullptr now that we have this logic.